### PR TITLE
ShowBizz Api based webApp

### PR DESCRIPTION
**Pull request by pinkmoon25**
# Description

In this Project we collaborated together to create ShowBizz, A webapp based on tv-shows using Tv-maze and Involvement API
we have followed this [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md )

here is the link to our presentation of the project [video](https://drive.google.com/file/d/1ft_SjhTG-vYD4-kCs8zGFygbFGfEaR9M/view?usp=sharing)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Store projects data in array of object
- [ ] Load projects data from array of objects for each project
- [ ] Create projects modal
- [ ] Style pojects modal
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [ ] Our code follows the style guidelines of this project
- [ ] We have performed a self-review of our code
- [ ] We have commented our code, particularly in hard-to-understand areas
- [ ] We have made corresponding changes to the documentation
- [ ] Our changes generate no new warnings
- [ ] We have added tests that prove our fix is effective or that our feature works
- [ ] New and existing unit tests pass locally with our changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>ShowBizz</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="./index.html" class="logo-link"><span class="logo show"><em>Show</em></span><span class="logo bizz"><em>Bizz</em></span>
+        </a>
+        <ul>
+          <li class="active-nav"><a href="#">Latest Shows</a></li>
```
Comment (Nemwel-Boniface) [inline src/index.html]: - [x] You have both done an amazing job with your project so far. However, kindly note that according to the provided wireframes in the project requirements, you are required to include a counter that counts the total number of movies that you currently have in your web application. Kindly include it as well as the unit tests for this counter as well. 

**This is how your project looks like:**

image

**This is the required header navbar section according to the project requirements:**

image

**Kindly include the counter as it is shown in the wireframe including its unit tests**

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @PinkMoon25 and  @Owusu-Desmond           :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

You have both done an amazing job with your Showbizz web application I really like the UI and the choice of colors that you all chose :fire: I really enjoyed doing a review of this project keep it up :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  
Excellent project demo video ✔️  


## ♻️ Required Changes ♻️

- [x] Kindly include the page header counter for all the movies listed in the web application including its unit tests and this project should be ready for approval :rocket: 

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle:
success
Hey @PinkMoon25, @Owusu-Desmond  :wave:

Great work on implementing the changes requested by the previous reviewer. Your project is complete. There is nothing else to say about your project. :+1:
Go ahead and merge it. :muscle:

## Highlights :1st_place_medal:
:heavy_check_mark: Following GitFlow
:heavy_check_mark: Following Linters
:heavy_check_mark: Implemented Requirements

### [Optional]

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by pinkmoon25** [comment]
Thank you @Nemwel-Boniface and @HAMMAS-SALEEM for the review :pray:

**Comment by HAMMAS-SALEEM** [comment]
#### Have a great day, @PinkMoon25 :+1:

**Pull request merged**